rllab/algos/batch_polopt.py

```diff
@@ -131,10 +133,14 @@ def train(self):
                 logger.log("saved")
                 logger.dump_tabular(with_prefix=False)
                 if self.plot:
-                    self.update_plot()
+                    # self.update_plot()
+                    # if self.pause_for_plot:
+                    #     input("Plotting evaluation run: Press Enter to "
+                    #               "continue...")
+                    rollout(self.env, self.policy, animated=True, max_path_length=self.max_path_length)
```

Comment:
Why do you switch the async plotting to the synchronized one? This is inappropriate since we need to keep the original features.

Reply:
Currently the window can only be drawn synchronously. I am currently working on being able to display it asynchronously.

Reply:
Did the mujoco_py upgrade break async plotting for Theano?

Reply:
That's correct. I'm currently facing a lot of bugs/issues with multi-threading/multi-processing for Theano, but synchronous plotting works.

Reply:
Okay. This is probably a good opportunity to fix it Once And For All.

Can you start by finding the root cause of the bug?

Reply:
Sure thing.